easybuild/framework/easyblock.py

```diff
@@ -3395,6 +3395,13 @@ def sanity_check_rpath(self, rpath_dirs=None, check_readelf_rpath=True):
                 self.log.debug(f"Sanity checking RPATH for files in {dirpath}")
 
                 for path in [os.path.join(dirpath, x) for x in os.listdir(dirpath)]:
+                    # skip the check for any symlinks that resolve to outside the installation directory
+                    if not is_parent_path(self.installdir, path):
+                        msg = (f"Skipping RPATH sanity check for {path}, since its absolute path resolves to outside "
+                               "the installation directory")
+                        self.log.debug(msg)
```

Comment:
log info and also mention path to install dir in there?

```suggestion
                        msg = (f"Skipping RPATH sanity check for {path}, since its absolute path resolves to outside "
                               f"the installation directory {self.installdir}")
                        self.log.info(msg)